readme.md

```diff
@@ -262,6 +262,18 @@ const validate = (input: unknown) => {
 };
 ```
 
+### Make `new Array()` return `unknown[]`
+
+With `new Array()` you can introduce `any`'s into your code:
+
+```ts
+import "@total-typescript/ts-reset/new-array";
+
+const arr = new Array(); // unknown[]
+
+const arr2 = []; // unknown[]
```

Comment:
This won't be possible! And I don't think should be possible - it currently infers as `never[]`, which is EVEN MORE correct than `unknown[]`

Reply:
Does it? When I try to test it, it comes back as any[].